internal/http/services/owncloud/ocdav/config/config.go

```diff
@@ -0,0 +1,86 @@
+package config
+
+import "github.com/cs3org/reva/v2/pkg/sharedconf"
+
+// Config holds the config options that need to be passed down to all ocdav handlers
+type Config struct {
+	Prefix string `mapstructure:"prefix"`
+	// FilesNamespace prefixes the namespace, optionally with user information.
+	// Example: if FilesNamespace is /users/{{substr 0 1 .Username}}/{{.Username}}
+	// and received path is /docs the internal path will be:
+	// /users/<first char of username>/<username>/docs
+	FilesNamespace string `mapstructure:"files_namespace"`
+	// WebdavNamespace prefixes the namespace, optionally with user information.
+	// Example: if WebdavNamespace is /users/{{substr 0 1 .Username}}/{{.Username}}
+	// and received path is /docs the internal path will be:
+	// /users/<first char of username>/<username>/docs
+	WebdavNamespace string `mapstructure:"webdav_namespace"`
+	SharesNamespace string `mapstructure:"shares_namespace"`
+	GatewaySvc      string `mapstructure:"gatewaysvc"`
+	Timeout         int64  `mapstructure:"timeout"`
+	Insecure        bool   `mapstructure:"insecure"`
+	// If true, HTTP COPY will expect the HTTP-TPC (third-party copy) headers
+	EnableHTTPTpc               bool                              `mapstructure:"enable_http_tpc"`
+	PublicURL                   string                            `mapstructure:"public_url"`
+	FavoriteStorageDriver       string                            `mapstructure:"favorite_storage_driver"`
+	FavoriteStorageDrivers      map[string]map[string]interface{} `mapstructure:"favorite_storage_drivers"`
+	Version                     string                            `mapstructure:"version"`
+	VersionString               string                            `mapstructure:"version_string"`
+	Edition                     string                            `mapstructure:"edition"`
+	Product                     string                            `mapstructure:"product"`
+	ProductName                 string                            `mapstructure:"product_name"`
+	ProductVersion              string                            `mapstructure:"product_version"`
+	AllowPropfindDepthInfinitiy bool                              `mapstructure:"allow_depth_infinity"`
```

Comment:
Are we sure we want this to be a breaking change? If we reverse the logic and call it `DisablePropfindDepthInfinity` we would have achieved the same, but would not have a breaking a change...

Reply:
According to the ticket... https://github.com/owncloud/ocis/issues/4188#issuecomment-1198112265

Reply:
IMO the change is only breaking for the CI. We should enable it for the CI and disable it for the default deployment.